test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
```

Comment:
You are not deploying "ruby-ex s2i image", you are deploying the ruby-ex application. s2i is the process of converting source to image. That's only a small part of what's going on.

It's more like: "A user can deploy the example Ruby application ruby-ex"